code/game/area/areas/lobotomy_corp.dm

```diff
@@ -59,6 +56,8 @@
 /area/department_main/extraction
 	name = "Extraction Main Room"
 
+/area/department_main/control
```

Comment:
Why would you move it down here?

Reply:
I was fucking about with control to try to get weather working